alacritty_terminal/src/grid/mod.rs

```diff
@@ -426,6 +420,11 @@ impl<T> Grid<T> {
         self.display_offset
     }
 
+    #[inline]
+    pub fn reset_display_offset(&mut self) {
+        self.display_offset = 0;
+    }
```

Comment:
Why is this a method? Just to have it accessibly publicly? Why not use the `scroll` method?

Reply:
> Why is this a method? Just to have it accessibly publicly?

I'm not sure that it should be public... In fact we just want to reset display offset to 0 but not change it to any value. So I create the method instead of being it publicity.

Reply:
But if we don't reset display offset regardless of whether or not vi-mode is enabled, I think we may need not to be it publicity or to create the method.

Reply:
> Why not use the scroll method?

Oh, I didn't notice that...

Reply:
That's what I'm here for. :smile: